scripts/thunderstorm-collector.sh

```diff
@@ -16,11 +16,16 @@ LOG_TO_CMDLINE=1
 
 # Thunderstorm Server
 THUNDERSTORM_SERVER="ygdrasil.nextron"
+THUNDERSTORM_PORT=8080
 USE_SSL=0
 ASYNC_MODE=1
 
+# Source
+HOSTNAME=$(hostname -f)
+echo "$HOSTNAME"
```

Comment:
This looks like debug output - either make it more verbose (like "Collecting from ..." or remove it.